public/view/overlay-controls/selected-admins-info.jsx

```diff
@@ -4,6 +4,11 @@ var _ = require('lodash');
 require('./selected-admins-info.css');
 
 var SelectedAdminsInfo = React.createClass({
+
+  commify: function( x ) {
+    return parseInt(x).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  },
```

Comment:
eslint says:
```
  8:20  error  There should be no spaces inside this paren  space-in-parens
  8:24  error  There should be no spaces inside this paren  space-in-parens
  9:12  error  Missing radix parameter                      radix
```

The [radix rule](http://eslint.org/docs/rules/radix) is a special bit of Javascript fun. Seems like it's less important for EC5, but I think still good practice to always pass a radix to `parseInt`.